src/MatchedLearnerApi/Program.cs

```diff
@@ -36,16 +30,19 @@ public static void Main(string[] args)
 
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>()
                 .ConfigureAppConfiguration((hostingContext, config) =>
                 {
-                    var environmentName = hostingContext.HostingEnvironment.EnvironmentName;
-                    config.SetBasePath(Directory.GetCurrentDirectory());
-                    config.AddJsonFile("appSettings.json", optional: false, reloadOnChange: false);
-                    config.AddJsonFile($"appSettings.{environmentName}.json", optional: true, reloadOnChange: false);
-                    config.AddEnvironmentVariables();
+                    config.AddAzureTableStorage(options =>
+                    {
+                        options.PreFixConfigurationKeys = false;
+                        options.ConfigurationKeys = new[] {MatchedLearnerApiConfigurationKeys.MatchedLearnerApiKey};
+                    });
+
+                    //NOTE: bellow option uses PreFixConfigurationKeys = true which means all the configurations are prefixed by "MatchedLearner:<key>"
+                    //config.AddAzureTableStorage(MatchedLearnerApiConfigurationKeys.MatchedLearnerApiKey);
```

Comment:
not sure this comment makes sense here anymore, especially as this option is set to false above. Also with this option set does MatchedLearnerApiConfigurationKeys class need to change?

Reply:
I left it for information purpose, in-case in future if anyone wan't to know why options.PreFixConfigurationKeys = false; was used